README.md

```diff
@@ -50,3 +51,6 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Answer
+My code first starts out with the initialization of the current route, which get us $n$. We then have the main while loop which runs $i$ times and for each time it runs it also has $n$ from the combination of the randomization and two opt swap and getting the new length. So, in total we have $i * n + n$ which can be simplified to $i * n$. So my worst case time complexity is $\Theta(i * n)$ where $n$ is the size of the input and $i$ is the number of iterations of randomization that the route goes through.
```

Comment:
Well, so how many randomizations would you go through in the worst case?

Reply:
It would go through 10,000,000 in the worst case so would it just be $\Theta(10000000n)$ simplified to $\Theta(n)$? After reviewing more of the slides that makes sense but it that just seems a little too easy.

Reply:
Is it really a constant? You're also checking whether the length improved, aren't you?

Reply:
Yeah I guess it's not because of the length improvement stuff. So would it be something in the ball park of $\Theta((i + x * r) * n)$ where x is continuations after the length hasn't improved and r is how many times we reset it. Or could it just be $\Theta((x * r) * n)$?

Reply:
Couldn't it improve every time you randomize it, so however many times you randomize it?

Reply:
Wouldn't that be $n!$?

Reply:
That sounds like it's going in the right direction.